### DISCUSSION TITLE
After updating all devices appear twice in Nora App and Google Home App

### DISCUSSION THREAD

**Discussion by Fragmike** [andrei-tatar/node-red-contrib-smartnora#196] [Q&A] [answered]
Updated my system and had to replace my old nora config. Made a new one and deleted old one. Now all devices appear twice...one is working, the other one is offline. Any Idea how to get rid of those ;-(
Greetings Michael

**Comment by Fragmike** [comment, marked as answer]
Ok...cleaning up all the groups sovled the Problem ;-)

**Reply by heisenberg2980** [reply]
Having the exact same issue now, how did you clean up the groups?

EDIT: found it, under the "Manage Groups" option in NORA website (https://smart-nora.eu/my-nora).
